augur/dates/__init__.py

```diff
@@ -121,6 +122,25 @@ def get_numerical_date_from_value(value, fmt=None, min_max_year=None):
         except InvalidDate as error:
             raise AugurError(str(error)) from error
         return [treetime.utils.numeric_date(d) for d in ambig_date]
+    if '/' in value:
+        # ISO interval
+        try:
+            start, end = aniso8601.parse_interval(value)
+        except ValueError as e:
+            raise AugurError(f"Invalid date {value!r}: {e}") from e
```

Comment:
_non-blocking question_

I'm slightly confused by the use of `AugurError` vs `InvalidDate`. Should `InvalidDate` be modified to be a child class of `AugurError` so it can be used directly?

Reply:
Good idea. Updated in 617ed588709d11172256b2a3ea814714f3174e07